test/Tryouts/Corax/IndexBuilder.cs

```diff
@@ -261,7 +262,7 @@ private PostingListWriter GetPostingListWriter(string field, string term)
             return postingList;
         }
 
-        private unsafe long GetStringId(string key, bool cache)
+        private unsafe long GetStringId(string key, bool cache, bool incrementFreq = false)
```

Comment:
Better to have a different method here, because this is very different. 
`Get` method should not be mutating state.